app/uk/gov/hmrc/platforminitiatives/services/PlatformInitiativesService.scala

```diff
@@ -73,7 +74,7 @@ class PlatformInitiativesService @Inject()(
       ),
       createUpgradeInitiative(
         initiativeName        = "Auth-client upgrade",
-        initiativeDescription = s"[CL250 Security upgrade required](https://catalogue.tax.service.gov.uk/dependencyexplorer/results?group=uk.gov.hmrc&artefact=auth-client&team=$teamName&flag=production&scope=compile&versionRange=%5B0.0.0%2C5.6.0%29&asCsv=false) | [Confluence](https://confluence.tools.tax.service.gov.uk/x/RgpxDw).",
+        initiativeDescription = s"[CL250 Security upgrade required](" + url"https://catalogue.tax.service.gov.uk/dependencyexplorer/results?group=uk.gov.hmrc&artefact=auth-client&team=$teamName&flag=production&scope=compile&versionRange=%5B0.0.0%2C5.6.0" + "\\)" + "&asCsv=false" + " ) | [Confluence](" + url"https://confluence.tools.tax.service.gov.uk/x/RgpxDw" + ").",
```

Comment:
CLARIFICATION: Wouldn't this double encode "%5B0.0.0%2C5.6.0" ?

Reply:
Shouldn't do, as this will interpolate the escape codes as well as the rest of the URI, insofar as it'll replace the codes with the relevant symbol. The only thing that doesn't work for the initiatives frontend is the closed bracket, as the markdown reader on the frontend considers that a way to close the URI, hence it's separately escaped. However, having said this, I'll change the escape codes to the relevant symbols for consistency.